### DISCUSSION TITLE
PanelMenu doesn't expand sub-items.

### DISCUSSION THREAD

**Discussion by Matthew404-fail** [primefaces/community#1311] [PrimeReact] [answered]
Hello!
I'm faced a problem with `PanelMenu`.
When I try to use `expanded` property with sub-item, it doesn't work.
![image](https://github.com/primefaces/community/assets/66127533/d8480f51-a037-470d-834d-20351a7f3bbb)

Example:
https://stackblitz.com/edit/oatqzb?file=src%2FApp.tsx

**Comment by melloware** [comment]
Please open a GitHub bug ticket with your reproducer please.

**Comment by melloware** [comment, marked as answer]
Bug reported: https://github.com/primefaces/primereact/issues/6054
